vitest.config.ts

```diff
@@ -4,9 +4,16 @@ import viteConfig from "./vite.config";
 export default mergeConfig(
   viteConfig,
   defineConfig({
+    build: {
+      sourcemap: "inline",
```

Comment:
is this set because separately generated sourcemap files interferes with coverage report generation?